closed/OpenJ9.gmk

```diff
@@ -228,11 +228,16 @@ ifeq (true,$(OPENJ9_ENABLE_DDR))
 $(call openj9_add_jdk_bin, java.base, j9ddr.dat)
 
 .PHONY : run-ddrgen
-$(OUTPUTDIR)/vm/j9ddr.dat : run-ddrgen
+$(OPENJ9_VM_BUILD_DIR)/j9ddr.dat : run-ddrgen
+
 run-ddrgen :
+ifeq (true,$(OPENJ9_ENABLE_CMAKE))
+	$(MAKE) -C $(OPENJ9_VM_BUILD_DIR) j9ddr
+else
 	export CC="$(CC)" CXX="$(CXX)" $(EXPORT_MSVS_ENV_VARS) \
 		&& $(MAKE) -C $(OUTPUTDIR)/vm/ddr -f run_omrddrgen.mk
-endif
+endif # ifeq (true,$(OPENJ9_ENABLE_CMAKE))
+endif # feq (true,$(OPENJ9_ENABLE_DDR))
```

Comment:
My preference is to just name the flag:
```
endif # OPENJ9_ENABLE_CMAKE
endif # OPENJ9_ENABLE_DDR
```
but if you want to repeat the condition, that's fine, just repeat all of it (`ifeq...` not `feq...`).